app/views/collaborators/_collaborator.html.erb

```diff
@@ -0,0 +1,18 @@
+ 
+ <div class="form-group">
+    <%# this should more properly use a variable set in your controller rather than User.all - for example you might want to limit the list of possible collaborators according to some condition %>
+    <% User.all.each do |user| %>
+      <% #if collaborator = user.collaborator_for(wiki) %>
+      <% if wiki.private? && user != wiki.user && !wiki.collaborators.include?(user) %>
```

Comment:
Review line 6 and compare to line 12. Line 12 is correctly checking if `user` is also a `collaborator_user`.